Jenkinsfile

```diff
@@ -56,6 +56,7 @@ pipeline {
       }
       steps {
         deploy("cardid", "test", null, true)
```

Comment:
this should be `..., false, false` as we do not want the deploy job to run tests after deploying regular cardid.